src/components/application_manager/src/rpc_handler_impl.cc

```diff
@@ -296,9 +296,9 @@ void RPCHandlerImpl::GetMessageVersion(
     }
     utils::SemanticVersion temp_version(major, minor, patch);
     if (temp_version.isValid()) {
-      message_version = (temp_version >= utils::rpc_version_5)
-                            ? temp_version
-                            : utils::base_rpc_version;
+      message_version = temp_version;
+    } else {
+      message_version = utils::base_rpc_version;
```

Comment:
@YarikMamykin Remove redundant `else`. You can do it like here: 
```c++
message_version = utils::base_rpc_version;
utils::SemanticVersion temp_version(major, minor, patch);
if (temp_version.isValid()) {
   message_version = temp_version;
}
```

Reply:
@AByzhynar , fixed in 75c4639d3ebb0d905b7414f87efef58b50cf99d7